src/Components/Search.jsx

```diff
@@ -15,11 +15,24 @@ class Search extends Component {
   }
 
   componentDidMount() {
-    //do fetch calls here
-    this.setState({
-      coloradoBeers: coloradoBeer,
-      coloradoBreweries: coloradoBreweries
-    })
+    fetch('https://whateverly-datasets.herokuapp.com/api/v1/coloradoBeer')
+      .then(response => response.json())
+      .then(triviaQuestions => {
```

Comment:
This should be coloradoBeer not triviaQuestions -> we get an error two lines down that coloradoBeer is not defined.